packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -1484,7 +1490,7 @@ class _LibraryRefColumn extends TreeColumnData<Reference> {
 
   @override
   String getTooltip(Reference dataObject) {
-    return dataObject.name;
+    return dataObject.name!;
```

Comment:
default to empty string instead of null asserting

Reply:
done